app/assets/javascripts/initializers/initializeHeroBannerClose.js

```diff
@@ -3,6 +3,7 @@
 function initializeHeroBannerClose() {
   let bannerWrapper = document.getElementById('hero-html-wrapper');
   let closeIcon = document.getElementById('js-hero-banner__x');
+  closeIcon.setAttribute('aria-label', 'Close campaign banner');
```

Comment:
Refactor (blocking): Move `setAttribute` into conditional check for `closeIcon`. I tested the happy path and clearly haven't had enough coffee. We need to check if the element exists before setting the attribute.

```js
  if (bannerWrapper && closeIcon) {
    closeIcon.setAttribute('aria-label', 'Close campaign banner');
...
```